### ISSUE TITLE
Register support for `.lnk` files in extension queries.

### ISSUE THREAD

**Issue by roke-julian-lockhart** [lowfront/vscode-windows-lnk#2]

**Comment by fronterior** [comment]
Does `vscode-windows-lnk` extension not work?

**Comment by roke-julian-lockhart** [comment]
> https://github.com/lowfront/vscode-windows-lnk/issues/2#issuecomment-1238189951

@lowfront, I'm not certain of what you mean. I thought that this was an extension for Visual Studio Code. Is it not?

**Comment by fronterior** [comment]
That's right. This is an extension that comes out when you search for [Windows Ink](https://marketplace.visualstudio.com/items?itemName=lowfront.vscode-windows-lnk) that is different from `open-lnk`. Is there an issue?

**Comment by roke-julian-lockhart** [comment]
> https://github.com/lowfront/vscode-windows-lnk/issues/2#issuecomment-1238376815

@lowfront, sorry, couldn't locate it. You should register that it supports `ext:lnk`, because searching for that string via `vscode` solely provides the aforementioned competitor. As an example, https://github.com/Jaimeloeuf/open-lnk/tree/b5e3c3d14b376689b5db386e0d95c19aa8c41aa6 is listed when searching for that query.

**Comment by fronterior** [comment]
Aha, I understood. It doesn't come out when I search it like that. I'll try to fix it so that it can be searched.
